RealestateApi/Validators/AddCustomerValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using FluentValidation;
+using RealEstateApi.Dto.Request;
+
+
+namespace RealEstateApi.Validators
+{
+    public class AddCustomerValidator : AbstractValidator<CustomerRequestDto>
+    {
+        public AddCustomerValidator()
+        {
+            RuleFor(x => x.Name).Length(5, 100).WithMessage("Name is required.");
+            RuleFor(x => x.Email).Length(8, 150).EmailAddress().WithMessage("A valid email is required");
+            RuleFor(x => x.Password).Length(5, 150).WithMessage("Password is required.");
```

Comment:
The task says that the validator fields should be required.

-> Name: Required. Min Length 5 max length 100
-> Email: Required. Min Length 8 Max length 150. Email format
-> Password: Required. Min Length: 5 max length: 150.

You must add ".NotEmpty()" to each of them.